tock/tock/urls.py

```diff
@@ -13,28 +11,40 @@
 
 urlpatterns = patterns(
     '',
-    url(r'^$', hours.views.ReportingPeriodListView.as_view(),
-        name='ListReportingPeriods'),
-    url(r'^reporting_period/', include("hours.urls.timesheets",
-        namespace="reportingperiod")),
-    url(r'^reports/', include("hours.urls.reports", namespace="reports")),
-    url(r'^employees/', include("employees.urls", namespace="employees")),
+    url(r'^$',
+        hours.views.ReportingPeriodListView.as_view(),
+        name='ListReportingPeriods'
+    ),
+    url(r'^reporting_period/', include(
+        'hours.urls.timesheets',
+        namespace='reportingperiod'
+    )),
+    url(r'^reports/', include(
+        'hours.urls.reports',
+        namespace='reports'
+    )),
+    url(r'^employees/', include(
+        'employees.urls',
+        namespace='employees'
+    )),
+    url(r'^projects/', include(projects.urls)),
 
     # TODO: version the API?
     url(r'^api/', include(api.urls)),
 
-    # project views
-    url(r'^projects/', include(projects.urls)),
-
-    # Uncomment the next line to enable the admin:
+    # Enable the Django admin.
     url(r'^admin/', include(admin.site.urls)),
 )
 
 
+# Enable Django Debug Toolbar only if in DEBUG mode.
 if settings.DEBUG:
     import debug_toolbar
```

Comment:
This might be over-optimizing, but I'm wondering if it might be better for this conditional to branch off `if 'debug_toolbar' in settings.INSTALLED_APPS` instead of `if settings.DEBUG`?  That way just in case someone has `DEBUG` enabled but doesn't have `debug_toolbar` installed, an exception won't be thrown. (For that matter, I'm uncertain why the debug toolbar doesn't add these routes itself, actually.)

Anyways, not a big deal either way!